spec/Section 6 -- Execution.md

```diff
@@ -513,9 +626,14 @@ CollectFields(objectType, selectionSet, variableValues, visitedFragments):
     - Append {selection} to the {groupForResponseKey}.
   - If {selection} is a {FragmentSpread}:
     - Let {fragmentSpreadName} be the name of {selection}.
-    - If {fragmentSpreadName} is in {visitedFragments}, continue with the next
-      {selection} in {selectionSet}.
-    - Add {fragmentSpreadName} to {visitedFragments}.
+    - If {fragmentSpreadName} provides the directive `@defer` and its {if}
+      argument is not {false} or is a variable in {variableValues} with the
```

Comment:
```suggestion
      argument is not {false} or a variable in {variableValues} with the
```

I think this reads better — otherwise it feels like it's saying "(is not false) or (is a variable with the value false)"

Reply:
I updated to use the same wording used for `includeDirective`